bedrock/mozorg/templates/mozorg/plugincheck.html

```diff
@@ -33,12 +33,6 @@
 
 {% block body_class %}sky{% endblock %}
 
-{% block optimizely %}
```

Comment:
Any reason why this block is being removed (as opposed to just turning off the waffle switch on prod?)